src/main/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/SampleSvcEndpoint.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.ons.ctp.response.collection.exercise.endpoint;
+
+import com.godaddy.logging.Logger;
+import com.godaddy.logging.LoggerFactory;
+import java.util.UUID;
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.ons.ctp.response.collection.exercise.lib.common.error.CTPException;
+import uk.gov.ons.ctp.response.collection.exercise.message.dto.SampleSummaryReadinessDTO;
+import uk.gov.ons.ctp.response.collection.exercise.repository.SampleLinkRepository;
+import uk.gov.ons.ctp.response.collection.exercise.service.CollectionExerciseService;
+
+@RestController
+@RequestMapping(value = "/sample", produces = "application/json")
+public class SampleSvcEndpoint {
+  private static final Logger log = LoggerFactory.getLogger(SampleSvcEndpoint.class);
+  private CollectionExerciseService collectionExerciseService;
+  private final SampleLinkRepository sampleLinkRepository;
+
+  @Autowired
+  public SampleSvcEndpoint(
+      CollectionExerciseService collectionExerciseService,
+      SampleLinkRepository sampleLinkRepository) {
+    this.collectionExerciseService = collectionExerciseService;
+    this.sampleLinkRepository = sampleLinkRepository;
+  }
+
+  @RequestMapping(value = "/summary-readiness", method = RequestMethod.POST)
+  public ResponseEntity<?> sampleSummaryReadiness(
```

Comment:
remove useage of generic wildcard

Reply:
Updates in separate PR https://github.com/ONSdigital/rm-collection-exercise-service/pull/321